content/articles/Factor-that-affects-the-performance-deep-learning-systems/index.md

```diff
@@ -0,0 +1,114 @@
+### INTRODUCTION
+Deep learning is a type of artificial intelligence that simulates how humans learn specific sorts of information. It's a branch of AI and machine learning. This technology has been used to create many self-driving cars and smart digital devices.
+ 
+When given raw data, a deep learning system chooses which attributes are most relevant on its own. Deep learning is carried out using [artificial neural networks](https://www.simplilearn.com/tutorials/deep-learning-tutorial/multilayer-perceptron) with multiple layers. 
+Deep Neural Networks (DNNs) are networks that conduct difficult operations such as representation and abstraction at each layer to make sense of images, music, and text. 
+Deep learning is the fastest-growing field in machine learning, and an increasing number of firms are using it to create new business models.
```

Comment:
These statements looks very irrelevant to the topic of this PR. Though, you have explained them in general, these statements doesn't introduce the reader to the topic of performance.

`Deep learning is the fastest-growing field in machine learning,` -> Everyone knows that, but how are you relating it to your topic.

`Deep Neural Networks (DNNs) are networks that conduct difficult operations such as representation and abstraction at each layer` -> This sentence looks highly abstract. Someone new to ML/DL, might not understand this in one go. Please try rephrasing it, and relate it to the performance side of DL